mono/tests/test-runner.cs

```diff
@@ -579,116 +581,12 @@ class TestInfo {
 
 	static void TryThreadDump (int pid, ProcessData data)
 	{
-		try {
-			TryGDB (pid, data);
-			return;
-		} catch {
-		}
-
-#if !FULL_AOT_DESKTOP && !MOBILE
-		/* LLDB cannot produce managed stacktraces for all the threads */
+#if !MOBILE
```

Comment:
Not 100% sure about this one. The original check seemed weird to me - why wouldn't we want to use `SIGQUIT` just because we're testing the FullAOT profile?